src/modules/sbstudio/plugin/model/global_settings.py

```diff
@@ -35,9 +71,19 @@ class DroneShowAddonGlobalSettings(AddonPreferences):
         name="Server URL",
         description=(
             "URL of a dedicated Skybrush Studio server if you are using a "
-            "dedicated server. Leave it empty to use the server provided for "
-            "the community for free"
+            "local server. Leave it empty to use the cloud server provided "
+            "for the community for free and for pro users with signed requests"
+        ),
+    )
+
+    gateway_url = StringProperty(
```

Comment:
As we discussed earlier, the user should not be able to override the gateway URL, otherwise it is much easier to stand in as a middleman batween the gateway and the Blender plugin.

However, we need to take care of the situation when the port we have chosen for the gateway is already occupied by some other service on the system. We will probably need to implement SSDP service discovery later to allow the port of the gateway to be dynamic.